lib/bcc_helper.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019 by Delphix. All rights reserved.
+ */
+
```

Comment:
Does this also need the GPL license?

Reply:
This could be any license we want.  GPL just to keep all consistent?

Reply:
Added.